user_guide/content/modules/red_hat_openshift_io_overview.adoc

```diff
@@ -5,15 +5,26 @@
 
 == Features
 
-{osio} offers features for seamless management of end-to-end application development. Members of distributed teams can use it to perform the following tasks without leaving the browser window:
+{osio} offers features to seamlessly manage end-to-end application development. Members of distributed teams can use it to perform the following tasks without leaving the browser window:
 
 * Project planning and issue tracking using your choice of Agile methodology.
 * Creating development environments and testing requirements.
-* Coding using an integrated development environment based on Eclipse Che.
-* Building, testing, deploying, and monitoring codebases using pipelines.
+* Coding using an integrated development environment based on Eclipse Che, which provides a workspace based on a containerized version of your application.
+* Development in this context-aware workspace provides the appropriate context for the developer and project, such as which branch, repository, language, and files are referenced.
+* Building, testing, deploying, and monitoring codebases using Continuous Integration pipelines that can be tailored to the requirements of a team or an individual.
+* Using integrated but customizable Jenkins build pipelines to facilitate rapid testing and environment promotion.
 * Interacting with integrated GitHub repositories.
+* Deploying applications packaged as a set of containers to OpenShift, which powers the underlying services, such as the Eclipse Che workspaces, Work Items, and Pipelines.
```

Comment:
which powers the underlying services>which is powered by the underlying services?